src/ServiceProviders/CIProviders/GithubActions/GithubActionsProvider.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace Pantheon\TerminusBuildTools\ServiceProviders\CIProviders\GithubActions;
+
+use Pantheon\TerminusBuildTools\ServiceProviders\CIProviders\BaseCIProvider;
+use Pantheon\TerminusBuildTools\ServiceProviders\CIProviders\CIProvider;
+use Pantheon\TerminusBuildTools\ServiceProviders\CIProviders\CIState;
+
+use Pantheon\TerminusBuildTools\Task\Ssh\PrivateKeyReciever;
+use Psr\Log\LoggerAwareInterface;
+
+use Pantheon\TerminusBuildTools\Credentials\CredentialClientInterface;
+
+use Pantheon\TerminusBuildTools\API\GitHub\GitHubAPITrait;
+
+use ParagonIE_Sodium_Compat;
```

Comment:
Is this provided by https://github.com/paragonie/sodium_compat? Where is this dependency pulled in?

Reply:
It was here but it was closed, I'm not sure why: https://github.com/pantheon-systems/terminus/pull/2134

Reply:
Added to this PR